doc/manual/profile.rst

```diff
@@ -211,7 +211,7 @@ Options for controlling the display of profile results
 :func:`Profile.print` has more options than we've described so far.
 Let's see the full declaration::
 
-    function print(io::IO = STDOUT, data = fetch(); format = :tree, C = false, combine = true, cols = tty_cols())
+    function print(io::IO = STDOUT, data = fetch(); format = :tree, C = false, combine = true, cols = tty_cols(), maxdepth = typemax(Int), sortedby = :filefuncline)
```

Comment:
Probably, but I'm not going to worry about that until the doc stuff settles out.